### PR TITLE
fix #62227 bauermedia.co.uk

### LINKED ISSUE #62227: bauermedia.co.uk
### Issue URL (Incorrect Blocking)
 [https://www.bauermedia.co.uk/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.bauermedia.co.uk%2F)
### Comment
Username: @benbristow
> Cookie warning overlay blocks page being used.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | EdgeChromium
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>I don't care about cookies<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>EasyList China, <br/>EasyList Lithuania, <br/>Dandelion Sprout's Nordic Filters

### PR DESCRIPTION
**Pull request by stanislav-atr**
#62227

### CODE REVIEW

**Comment by Alex-302** [comment]
fix for platforms without CSS support

**Review by Alex-302 — CHANGES_REQUESTED** [review]
fix for platforms without CSS support

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -752,6 +752,9 @@ drimsim.com##.cookie[data-ix="cookie-close"]
 engie-energie.nl#$##cookieModal { display: none!important; }
 engie-energie.nl#$#.reveal-modal-bg { display: none!important; }
 engie-energie.nl#$#body { overflow: visible!important; }
+bauermedia.co.uk#$#.modal-backdrop { display: none !important; }
+bauermedia.co.uk#$#body { overflow: visible !important; }
+||bauermedia.co.uk/scripts/vendor.js
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:757]: you just break the site. Exclude required element for content blockers.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_specific.txt:757]: What does it break? Everything works fine on my end.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**